src/Neo/Hardfork.cs

```diff
@@ -15,6 +15,7 @@ public enum Hardfork : byte
     {
         HF_Aspidochelone,
         HF_Basilisk,
-        HF_Cockatrice
+        HF_Cockatrice,
+        HF_Domovoi
```

Comment:
Suggestions on name are welcomed.

Reply:
I am not familier with any of them, LOL, LGTM

Reply:
Same as Jimmy